pkgs/applications/emulators/yuzu/mainline.nix

```diff
@@ -120,6 +121,8 @@ in stdenv.mkDerivation {
     "-DENABLE_QT_TRANSLATION=ON"
 
     # use system libraries
+    # NB: "external" here means "from the externals/ directory in the source",
+    # so "off" means "use system"
```

Comment:
That's impressively misleading. Thanks for documenting!

Reply:
I honestly added that note because every time I find this code I get very confused and have to go check again.